### ISSUE TITLE
Got an error while attempting to run shell on deployment (Minishift)

### ISSUE THREAD

**Issue by kaerimichi** [telepresenceio/telepresence#762]
### What were you trying to do?

Swap deployment 'hello-world' on Minishift

### What did you expect to happen?

I expect to access the deployment shell

### What happened instead?

The command returned an error

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '--swap-deployment', 'hello-world', '--run-shell']`
Version: `0.92`
Python version: `3.7.0 (default, Jul 23 2018, 20:22:55) 
[Clang 9.1.0 (clang-902.0.39.2)]`
kubectl version: `Client Version: v1.11.2 // Server Version: v1.10.0+b81c8f8`
oc version: `oc v3.10.0+dd10d17 // kubernetes v1.10.0+b81c8f8 // features: Basic-Auth //  // Server https://192.168.64.3:8443 // openshift v3.10.0+ddf284b-16 // kubernetes v1.10.0+b81c8f8`
OS: `Darwin DOT246MAC.local 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/telepresence/0.92/libexec/lib/python3.7/site-packages/telepresence/cli.py", line 129, in crash_reporting
    yield
  File "/usr/local/Cellar/telepresence/0.92/libexec/lib/python3.7/site-packages/telepresence/main.py", line 43, in main
    runner.kubectl = KubeInfo(runner, args)
  File "/usr/local/Cellar/telepresence/0.92/libexec/lib/python3.7/site-packages/telepresence/startup.py", line 135, in __init__
    self.in_local_vm = self._check_if_in_local_vm(runner)
  File "/usr/local/Cellar/telepresence/0.92/libexec/lib/python3.7/site-packages/telepresence/startup.py", line 167, in _check_if_in_local_vm
    if ip and ip in runner.kubectl.server:
AttributeError: 'NoneType' object has no attribute 'server'

```

Logs:

```
   0.0 TEL | Telepresence 0.92 launched at Wed Aug 29 10:48:54 2018
   0.0 TEL |   /usr/local/bin/telepresence --swap-deployment hello-world --run-shell
   0.0 TEL | Platform: darwin
   0.0 TEL | Python 3.7.0 (default, Jul 23 2018, 20:22:55)
   0.0 TEL | [Clang 9.1.0 (clang-902.0.39.2)]
   0.0 TEL | [1] Running: uname -a
   0.0   1 | Darwin DOT246MAC.local 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64
   0.1 TEL | BEGIN SPAN main.py:41(main)
   0.1 TEL | BEGIN SPAN startup.py:57(__init__)
   0.1 TEL | [2] Capturing: kubectl version --short
   0.6 TEL | [3] Capturing: kubectl config current-context
   0.7 TEL | [4] Capturing: kubectl config view -o json
   0.8 TEL | Command: oc 1.11.2
   0.8 TEL | Context: myproject/192-168-64-3:8443/developer, namespace: myproject, version: 1.10.0+b81c8f8
   0.8 TEL | [5] Capturing: minishift ip
```

**Comment by ark3** [comment]
Sorry about the crash! Our testing around OpenShift and MiniShift is lacking. It looks like we broke something early in startup. Unfortunately, I don't have a workaround for you at the moment.

**Comment by FedericoPonzi** [comment]
Same for me! Is there maybe an older working version of telepresence we can try out?

**Comment by tellisnz** [comment]
I think this was fixed by #797

**Comment by kaerimichi** [comment]
Thanks, @tellisnz. I can confirm that this isn't happening anymore in version 0.94.
